systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -118,8 +118,13 @@ public void installUsingHelm() {
      */
     public void createClusterOperatorNamespace() {
         if (KubeResourceManager.get().kubeClient().namespaceExists(clusterOperatorConfiguration.getNamespaceName())) {
-            Namespace coNamespace = KubeResourceManager.get().kubeClient().getClient().namespaces().withName(clusterOperatorConfiguration.getNamespaceName()).get();
-            KubeResourceManager.get().deleteResource(coNamespace);
+            if (!clusterOperatorConfiguration.shouldSkipNamespaceDeletion()) {
```

Comment:
Just a nit, but flip the condition to avoid using negation inside if?

Reply:
yeah I can change it to `dontSkipNamespaceDeletion` and negate it inside that method.

Reply:
actually, I will name it `shouldDeleteNamespace` and flip the overall boolean value, that will be better.